### ISSUE TITLE
Histograms and variance of variables

### ISSUE THREAD

**Issue by AmandaFranklinRyan** [AmandaFranklinRyan/SupervisedMachineLearning#3]
Here is a histogram showing the variables with less than 90% missing values and 95% correlation:

![All Histograms](https://github.com/AmandaFranklinRyan/SupervisedMachineLearning/assets/26112455/4efc9b95-4bd5-4d67-a98f-b8672d17face)

This plot shows the features with the lowest variance:

![Lowest Variance Bar](https://github.com/AmandaFranklinRyan/SupervisedMachineLearning/assets/26112455/b6475f6d-abc1-445d-832c-084e356a648a)

On this basis I think it makes sense to drop lat, lon, avg_bauperiode and month. What do you think?

**Comment by linanita22** [comment]
I agree with you to drop lat, lon and month. But why do you think that avg_bauperiode has such a low variance? Looking at the plot above, it shows that we do have different values. Also, when looking at my data profiling file, we have 97.8% of the values. I therefore would include it in the model....
